src/index.html

```diff
@@ -7,7 +7,58 @@
     <link rel="stylesheet" href="./index.scss">
   </head>
   <body>
-    <h1>Hello, email client!</h1>
+    <div class="wrapper">
+      <div class="email-client">
+        <div class="email-list-container">
+          <h2 class="email-header">Inbox</h2>
+          <ul class="email-list">
+            <li class="email-item">
+              <div class="sender-details">
+                <p>Email sender</p>
+                <span>Timestamp</span>
+              </div>
+              <p class="email-subject">Email1 subject</p>
+              <p>Email snippet</p>
+            </li>
```

Comment:
Should we include an `<a>` tag in here somewhere to nudge people away from the inaccessible `<div onclick=...>`

Reply:
Agree 💯 but how about `button` instead since we aren't really going to a new page?

Reply:
👏 Good catch

Reply:
I guess I had imagined that we would eventually use url routing, and that it would be a semantic link. I consider `a` tags semantically correct for idempotent requests (like GET) and buttons more appropriate for requests that make changes. But you can argue it either way...